Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -256,6 +256,16 @@ WI.CSSManager = class CSSManager extends WI.Object
             return WI.unlocalizedString("::scrollbar-corner");
```

Comment:
@dcrousso @patrickangle Fwiw, I noticed these show the wrong strings. The scrollbar stuff should all be prefixed with -webkit-, I didn't touch it here, since there's likely more stuff to change.

Reply:
interesting.  yeah probably should change it.  feel free to do a followup fix if you'd like :)

Reply:
Not sure to which extent I should change things without breaking compat nor how to test, so I'll leave it to someone else